osu.Game/Users/UserActivity.cs

```diff
@@ -25,9 +25,14 @@ public class ChoosingBeatmap : UserActivity
             public override string Status => "Choosing a beatmap";
         }
 
-        public class MultiplayerGame : UserActivity
+        public class MultiplayerGame : SoloGame
         {
-            public override string Status => "Playing with others";
+            public MultiplayerGame(BeatmapInfo beatmap, RulesetInfo ruleset)
+                : base(beatmap, ruleset)
+            {
+            }
+
+            public override string Status => $@"{base.Status} with others";
```

Comment:
This won't work for rulesets that don't override `PlayingVerb`.

https://github.com/ppy/osu/blob/78ac092147d994aa8a832584a66ad4df42c45c1b/osu.Game/Rulesets/Ruleset.cs#L226-L229

It's not a huge deal for now as only the built-in ones are permitted and those all override it, but the potential of having this return `Playing solo with others` indicates that this may not be the optimal way to do this.

(Also, as an aside, this is a bit of an interesting one wrt localisation - should the status returned to discord be localised according to the user's selected language, or not?)

Reply:
Should `Ruleset.PlayingVerb` be changed to "Playing" then? It is expected that many custom rulesets may want to override `PlayingVerb`.

Also, I don't think localization is necessary as many other games that use Rich Presence (and Spotify too if that counts) use English.

Reply:
I guess that could work? No strong opinion from me.

Reply:
Might as well make it `abstract` and avoid arbitrary strings?

Reply:
Breaking ruleset API change, so I'm not sure I agree.